<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>994</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#header {
	background-color: #004d99;
	padding-left: 10px;
	margin-left: 10px;
	margin-right: 10px;
}

#labelMainMenuTitle {
	font-size: 32px;
	font-weight: bold;
	margin-bottom: 10px;
}

#labelSelection {
	font-size: 16px;
}

#title {
	color: white;
	font-weight: bold;
	font-size: 36px;
}

#btnBack {
	background-color: #4db8ff;
	border-radius: 20px;
	color: white;
	margin-top: 10px;
	font-size: 24px;
	height: 50px;
}

#btnBack:hover {
	background-color: #1AA4FF;
}

#menuFrame {
	padding: 10px 0px;
	border: 2px solid #0073e6;
	border-radius: 20px;
	margin-right: 10px;
	margin-left: 3px;
}

#menuFrame QPushButton {
	color: #ffffff;
	background-color:  #004d99;
	border-radius: 15px;
	font-weight: bold;
	font-size: 24px;
	height: 80px;
	padding: 10px 16px;
	margin-bottom: 10px;
}

#menuFrame QPushButton:hover {
	background-color: #0067CC;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="header">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btnBack">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>10</y>
            <width>131</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
          <property name="icon">
           <iconset resource="../qrc/main_menu_rc.qrc">
            <normaloff>:/images/assets/arrow-left.svg</normaloff>:/images/assets/arrow-left.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="logo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qrc/main_menu_rc.qrc">:/images/assets/logo-white.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="title">
         <property name="text">
          <string>PROJECT: WattWise</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="menuFrame">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelMainMenuTitle">
         <property name="text">
          <string>Electrical Engineering and Professional Subject (EEPS)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QFrame" name="subcategMainFrame">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="btnElectricCircuits">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Electric Circuits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnElectronicTheoryCircuits">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Electronic Theory &amp;&amp; Circuits</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnEnergyConversion">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Energy Conversion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPowerTransmissionDistribution">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Power Transmission &amp;&amp; Distribution</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnInstrumentationMeasurement">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Instrumentation &amp;&amp; Measurement</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCircuitLineProtection">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Circuit &amp;&amp; Line Protection</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btnControlSystem">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Control System</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPrinciplesCommunication">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Principles of Communication</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnElectricalEquipment">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Electrical Equipment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnComponentsDevices">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Components Devices</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnElectricalMachines">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Electrical Machines</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="btnPowerPlant">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Power Plant</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnIllumination">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Illumination</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuildingWiring">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Building Wiring</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnElectronicPowerEquipment">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Electronic Power Equipment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnElectricSystems">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Electric Systems</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="labelSelection">
         <property name="text">
          <string>Please choose a subcategory</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/main_menu_rc.qrc"/>
 </resources>
 <connections/>
</ui>
